### ISSUE TITLE
Cannot deploy to cluster node - failed to execute cluster config script 

### ISSUE THREAD

**Issue by dcorrigangoempyrean** [codership/galera-manager-support#64]
I'm trying to deploy to a cluster node running Ubuntu 22.04, Galera 4 and MySQL 8

Jul 25, 2023 19:27:01 | galera-manager | Got an error and attepts = 0
Jul 25, 2023 19:27:01 | galera-manager | failed to execute cluster config script (RunScriptWithConn): command failed (stepName=__step_no_001, commandId=1, commandType=IncludeCommand): command failed (stepName=import_influxdata_gpg, commandId=10, commandType=ExecCommand): Process exited with status 2
Jul 25, 2023 19:27:01 | galera-manager | root@node1# echo -n "test"
Jul 25, 2023 19:27:01 | stdout         | test
Jul 25, 2023 19:27:01 | galera-manager | Saving Galera Manager node data

msg: failed to execute cluster config script (RunScriptWithConn)
stackTrace:
  - "github.com/codership/galera-manager/pkg/internal/sshcmd.(*Host).RunScriptWithConn\n\t/go/pkg/internal/sshcmd/executor.go:115"
  - "github.com/codership/galera-manager/pkg/internal/sshcmd.(*Host).RunScript\n\t/go/pkg/internal/sshcmd/executor.go:171"
  - "github.com/codership/galera-manager/pkg/internal/mgmt/units.(*Node).Install\n\t/go/pkg/internal/mgmt/units/node.go:353"
  - "github.com/codership/galera-manager/pkg/internal/mgmt.(*Nodes).Create.func2\n\t/go/pkg/internal/mgmt/nodes.go:115"
  - "github.com/codership/galera-manager/pkg/internal/jobs.(*Processor).Execute.func1\n\t/go/pkg/internal/jobs/processor.go:90"
  - "runtime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1594"
wrapped:
  msg: Process exited with status 2
Any ideas?

I was able to deploy to another cluster with some issues, but same OS and Galera, MySQL versions. Essentially no difference.

How can I get more detail on what is actually failing and why ?

**Comment by dcorrigangoempyrean** [comment]
msg: failed to execute cluster config script (RunScriptWithConn)
stackTrace:
  - "github.com/codership/galera-manager/pkg/internal/sshcmd.(*Host).RunScriptWithConn\n\t/go/pkg/internal/sshcmd/executor.go:115"
  - "github.com/codership/galera-manager/pkg/internal/sshcmd.(*Host).RunScript\n\t/go/pkg/internal/sshcmd/executor.go:171"
  - "github.com/codership/galera-manager/pkg/internal/mgmt/units.(*Node).Install\n\t/go/pkg/internal/mgmt/units/node.go:353"
  - "github.com/codership/galera-manager/pkg/internal/mgmt.(*Nodes).Create.func2\n\t/go/pkg/internal/mgmt/nodes.go:115"
  - "github.com/codership/galera-manager/pkg/internal/jobs.(*Processor).Execute.func1\n\t/go/pkg/internal/jobs/processor.go:90"
  - "runtime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1594"
wrapped:
  msg: Process exited with status 2

**Comment by adnaniz** [comment]
I have the same issue.. 

`Jul 25, 2023 19:27:01 | galera-manager | Got an error and attepts = 0
Jul 25, 2023 19:27:01 | galera-manager | failed to execute cluster config script (RunScriptWithConn): command failed (stepName=__step_no_001, commandId=1, commandType=IncludeCommand): command failed (stepName=import_influxdata_gpg, commandId=10, commandType=ExecCommand): Process exited with status 2
Jul 25, 2023 19:27:01 | galera-manager | root@node1# echo -n "test"
Jul 25, 2023 19:27:01 | stdout | test
Jul 25, 2023 19:27:01 | galera-manager | Saving Galera Manager node data`

how to solve the issue..

**Comment by dcorrigangoempyrean** [comment]
Did you retry?  Every single time I added a node, I had to do it twice. 

Also, if I tried to install galera-manager ahead of time, it screwed everything up and the deployment would never finish. 
I ultimately abandoned use of the product.  It's not mature enough for a prime time solution. I was able to collect all of the same statistics through Zabbix anyway. The only advantage GM has is cluster control, which I don't really need.
